packages/@aws-cdk/aws-scheduler-targets-alpha/test/integ.lambda-invoke.js.snapshot/aws-cdk-schedule.template.json

```diff
@@ -108,7 +108,7 @@
      "Mode": "OFF"
     },
     "ScheduleExpression": "rate(1 minute)",
-    "ScheduleExpressionTimezone": "Etc/UTC",
+    "State": "ENABLED",
```

Comment:
@kaizencc  that was the change in the template after your PR (https://github.com/aws/aws-cdk/pull/27105). I am not sure that we should pass the `State` property, by default it's `ENABLED` anyway. Are there any recommendations on passing default values?

Reply:
It's fine to pass in the default in this case